pkgs/servers/home-assistant/default.nix

```diff
@@ -0,0 +1,55 @@
+#{ stdenv, buildPythonApplication, fetchPypi, callPackage
+#, requests, pyyaml, pytz, pip, jinja2, voluptuous, typing, aiohttp, yarl, async-timeout, chardet, astral, certifi
+#, sqlalchemy, netdisco, xmltodict, user-agents
+{ stdenv, python3
+, extraPackages ? x: [] }:
+
+let
+  python = python3.override {
+    packageOverrides = self: super: {
+      yarl = super.yarl.overridePythonAttrs (oldAttrs: rec {
+        name = "${oldAttrs.pname}-${version}";
+        version = "0.18.0";
+        src = oldAttrs.src.override {
+          inherit version;
+          sha256 = "11j8symkxh0ngvpddqpj85qmk6p70p20jca3alxc181gk3vx785s";
+        };
+      });
+      aiohttp = super.aiohttp.overridePythonAttrs (oldAttrs: rec {
+        name = "${oldAttrs.pname}-${version}";
+        version = "2.3.7";
+        src = oldAttrs.src.override {
+          inherit version;
+          sha256 = "0fzfpx5ny7559xrxaawnylq20dvrkjiag0ypcd13frwwivrlsagy";
+        };
+      });
+    };
+  };
+  hass-frontend = python.pkgs.callPackage ./frontend.nix { };
+in with python.pkgs; buildPythonApplication rec {
+  pname = "homeassistant";
+  version = "0.61.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1r0hwp7ds9c6af73b8jpwhay62n15zabwks9fyf7hxnxyz3ljl4p";
+  };
+
+  propagatedBuildInputs = [
+    # From setup.py
+    requests pyyaml pytz pip jinja2 voluptuous typing aiohttp yarl async-timeout chardet astral certifi
+    # From running hass
+    hass-frontend sqlalchemy netdisco xmltodict user-agents
+  ] ++ extraPackages python.pkgs;
```

Comment:
I suggest having `extraPackages python.pkgs` either in the `let` expression assigned to something, or have parentheses here around it, just for clarity. I don't know what is better. What do you think?